### ISSUE TITLE
[Bug] tts-server hangs

### ISSUE THREAD

**Issue by chigkim** [idiap/coqui-ai-TTS#24]
### Describe the bug

If I type tts-server --model_name, it just hangs with no output. NO error no nothing.
Even tts-server -h does this.
The tts-server from coqui-ai/TTS repo works.

### To Reproduce

```
git clone https://github.com/idiap/coqui-ai-TTS
cd coqui-ai-TTS
pip install -e .[server]
tts-server
```

### Expected behavior

It should start the server.

### Logs

```shell
If I interrupt with control+c after a while I get this:

(.venv) D:\code\python\coqui-ai-TTS>tts-server
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "D:\code\python\coqui-ai-TTS\.venv\Scripts\tts-server.exe\__main__.py", line 4, in <module>
  File "D:\code\python\coqui-ai-TTS\TTS\server\server.py", line 20, in <module>
    from TTS.utils.synthesizer import Synthesizer
  File "D:\code\python\coqui-ai-TTS\TTS\utils\synthesizer.py", line 12, in <module>
    from TTS.tts.configs.vits_config import VitsConfig
  File "D:\code\python\coqui-ai-TTS\TTS\tts\configs\vits_config.py", line 5, in <module>
    from TTS.tts.models.vits import VitsArgs, VitsAudioConfig
  File "D:\code\python\coqui-ai-TTS\TTS\tts\models\vits.py", line 34, in <module>
    from TTS.tts.utils.text.characters import BaseCharacters, BaseVocabulary, _characters, _pad, _phonemes, _punctuations
  File "D:\code\python\coqui-ai-TTS\TTS\tts\utils\text\__init__.py", line 1, in <module>
    from TTS.tts.utils.text.tokenizer import TTSTokenizer
  File "D:\code\python\coqui-ai-TTS\TTS\tts\utils\text\tokenizer.py", line 6, in <module>
    from TTS.tts.utils.text.phonemizers import DEF_LANG_TO_PHONEMIZER, get_phonemizer_by_name
  File "D:\code\python\coqui-ai-TTS\TTS\tts\utils\text\phonemizers\__init__.py", line 17, in <module>
    ESPEAK_LANGS = list(ESpeak.supported_languages().keys())
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\code\python\coqui-ai-TTS\TTS\tts\utils\text\phonemizers\espeak_wrapper.py", line 235, in supported_languages
    for count, line in enumerate(_espeak_exe(_DEF_ESPEAK_LIB, args, sync=True)):
                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\code\python\coqui-ai-TTS\TTS\tts\utils\text\phonemizers\espeak_wrapper.py", line 80, in _espeak_exe
    res2 = list(res)
           ^^^^^^^^^
KeyboardInterrupt
^C
```

### Environment

```shell
{
    "CUDA": {
        "GPU": [],
        "available": false,
        "version": null
    },
    "Packages": {
        "PyTorch_debug": false,
        "PyTorch_version": "2.3.0+cpu",
        "TTS": "0.23.1",
        "numpy": "1.26.4"
    },
    "System": {
        "OS": "Windows",
        "architecture": [
            "64bit",
            "WindowsPE"
        ],
        "processor": "Intel64 Family 6 Model 94 Stepping 3, GenuineIntel",
        "python": "3.11.9",
        "version": "10.0.19045"
    }
```

**Comment by chigkim** [comment]
I clone the fresh repo,checkout the commit, created vem from scratch, pip install, and ran the server.
There's still no output, and I can't open the server from the browser.
I waited 10 minutes to be sure. 

```
git clone https://github.com/idiap/coqui-ai-TTS
cd coqui-ai-TTS
git checkout a0f281e4
py -3.11 -m venv .venv
.venv\Scripts\activate
pip install -e .[server]
tts-server
```

**Comment by eginhard** [comment]
When interrupting with Control+C, is it always stuck in the same part?

**Comment by chigkim** [comment]
Yes, it's always seems to get stuck at the same place.
It's not just server. Just tts has this problem as well.
I just pulled the latest commit, and ran `pip install -e .`, and still having the same problem.

```
(.venv) D:\code\python\coqui-ai-TTS>;tts --list_models
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "D:\code\python\coqui-ai-TTS\.venv\Scripts\tts.exe\__main__.py", line 7, in <module>
  File "D:\code\python\coqui-ai-TTS\TTS\bin\synthesize.py", line 345, in main
    from TTS.api import TTS
  File "D:\code\python\coqui-ai-TTS\TTS\api.py", line 11, in <module>
    from TTS.utils.synthesizer import Synthesizer
  File "D:\code\python\coqui-ai-TTS\TTS\utils\synthesizer.py", line 12, in <module>
    from TTS.tts.configs.vits_config import VitsConfig
  File "D:\code\python\coqui-ai-TTS\TTS\tts\configs\vits_config.py", line 5, in <module>
    from TTS.tts.models.vits import VitsArgs, VitsAudioConfig
  File "D:\code\python\coqui-ai-TTS\TTS\tts\models\vits.py", line 34, in <module>
    from TTS.tts.utils.text.characters import BaseCharacters, BaseVocabulary, _characters, _pad, _phonemes, _punctuations
  File "D:\code\python\coqui-ai-TTS\TTS\tts\utils\text\__init__.py", line 1, in <module>
    from TTS.tts.utils.text.tokenizer import TTSTokenizer
  File "D:\code\python\coqui-ai-TTS\TTS\tts\utils\text\tokenizer.py", line 6, in <module>
    from TTS.tts.utils.text.phonemizers import DEF_LANG_TO_PHONEMIZER, get_phonemizer_by_name
  File "D:\code\python\coqui-ai-TTS\TTS\tts\utils\text\phonemizers\__init__.py", line 29, in <module>
    ESPEAK_LANGS = list(ESpeak.supported_languages().keys())
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\code\python\coqui-ai-TTS\TTS\tts\utils\text\phonemizers\espeak_wrapper.py", line 235, in supported_languages
    for count, line in enumerate(_espeak_exe(_DEF_ESPEAK_LIB, args, sync=True)):
                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\code\python\coqui-ai-TTS\TTS\tts\utils\text\phonemizers\espeak_wrapper.py", line 80, in _espeak_exe
    res2 = list(res)
           ^^^^^^^^^
KeyboardInterrupt
^C
```

**Comment by eginhard** [comment]
@chigkim I've merged a fix into the `dev` branch, it seems you're not using the latest code yet. Try:
```bash
git checkout dev
git pull origin dev --rebase
```
Otherwise it will also be released in version 0.24.1 shortly.

**Comment by chigkim** [comment]
Ah, sorry, I thought it was merged to main. Dev works! Thanks for the fix!
